compiler/compiler/src/main/java/org/robovm/compiler/StrippedMethodCompiler.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 Achrouf Abdenour <achroufabdenour@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/gpl-2.0.html>.
+ */
+
+package org.robovm.compiler;
+
+import org.robovm.compiler.config.Config;
+import org.robovm.compiler.llvm.Function;
+import org.robovm.compiler.llvm.Unreachable;
+
+import soot.SootMethod;
+
+public class StrippedMethodCompiler extends AbstractMethodCompiler {
```

Comment:
What is this class doing?
It has no documentation/javadoc.

Reply:
in whole program mode to  not waste time we create only a small method with an unrechable instruction  for  an unreachable method  later this weak method will be replaced by an alias method pointing to "strippedMethod" function,this avoid generating unreachable methods to get less time whe compiling in whole program mode